autoextract/aio/retry.py

```diff
@@ -104,6 +117,7 @@ def __init__(self):
         self.stop_on_throttling_error = stop_never
         self.stop_on_network_error = stop_after_delay(15 * 60)
```

Comment:
I think we should just keep a single stop for these three `network_error`, `server_error` and `query_error`. 

Something like `self.stop_after_some_time =  stop_after_delay(15 * 60)`

As it is now is confusing

Reply:
Sure, this is a good time to refactor it!

Reply:
Done!